library/alloc/src/bstr.rs

```diff
@@ -264,6 +288,7 @@ impl<'a> From<ByteString> for Cow<'a, ByteStr> {
 #[cfg(not(test))] // https://github.com/rust-lang/rust/issues/135100
 #[unstable(feature = "bstr", issue = "134915")]
 impl<'a> From<&'a ByteString> for Cow<'a, ByteStr> {
+    /// Wrap `ByteString` as byte str in `Cow::Borrowed`.
```

Comment:
This is inconsistent with the next `&'a ByteStr` documentation. I think the owned documentations should be similar to the other owned documentations, and the borrowed documentations should be similar to the other borrowed documentations